components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-overview/overview-trend/overview-trend.component.ts

```diff
@@ -238,13 +249,23 @@ export class OverviewTrendComponent implements OnChanges, OnDestroy  {
 
     update.select('.status-dot.failed')
       .attr('r', d => {
+        if (d.failed === d.passed && d.failed === d.skipped && d.failed === d.waived) { return 10; }
         if (d.failed === d.passed && d.failed === d.skipped) { return 8; }
-        if (d.failed === d.passed || d.failed === d.skipped) { return 6; }
+        if (d.failed === d.passed && d.failed === d.waived) { return 8; }
+        if (d.failed === d.skipped && d.failed === d.waived) { return 8; }
+        if (d.failed === d.passed || d.failed === d.skipped || d.failed === d.waived) { return 6; }
         return 4;
       });
 
     update.select('.status-dot.passed')
-      .attr('r', d => d.passed === d.skipped ? 6 : 4);
+      .attr('r', d => {
+        if (d.passed === d.skipped && d.passed === d.waived) { return 8; }
+        if (d.passed === d.skipped || d.passed === d.waived) { return 6; }
+        return 4;
```

Comment:
Out of curiosity - what is the reason here for returning 10, 8, 6, or 4?

Reply:
Hi @SEAjamieD ,
As there are 4 circles now (max value increased from 8 to 10), each circle will take 2 units and base circle default was 4. 
That makes 4+2+2+2 = 10
if there are 3 circles then, 4+2+2 = 8
if there are 2 circles then, 4+2 = 6
if its only 1 circle then 4

Reply:
Thanks @vivekshankar1!  I appreciate the education 👍

Reply:
@vivekshankar1 I want to urge a follow-up that makes the code much less obfuscated. As it is, it is very challenging to figure out what it is doing, but since you have already done the figuring, you are at an ideal point to explain it. 😁 
I would consider replacing the magic numbers with named constants. Also, it needs some comments. I would start with explaining what the block of `update.select('.status-dot.failed')` is supposed to be doing, as I have no idea.
(Thanks for pointing that out, @SEAjamieD )

Reply:
Hi @msorens , i have added comments and the requested changes. If this is good i can send another PR
https://github.com/vivekshankar1/automate/blob/02ffd2f502532364816990a746087b3afb5fcb4a/components/automate-ui/src/app/pages/%2Bcompliance/%2Breporting/%2Breporting-overview/overview-trend/overview-trend.component.ts#L26-L31

https://github.com/vivekshankar1/automate/blob/02ffd2f502532364816990a746087b3afb5fcb4a/components/automate-ui/src/app/pages/%2Bcompliance/%2Breporting/%2Breporting-overview/overview-trend/overview-trend.component.ts#L257-L301